### ISSUE TITLE
Doesn't work on Windows

### ISSUE THREAD

**Issue by A1rPun** [A1rPun/formatree-cli#1]
It does on Linux but Windows throws this error. Working on it :)
```
C:\Program Files\nodejs\node_modules\formatree-cli\node_modules\formatree\index.js:29
        )
        ^
SyntaxError: Unexpected token )
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Program Files\nodejs\node_modules\formatree-cli\cli.js:3:19)
    at Module._compile (module.js:570:32)
```

**Comment by A1rPun** [comment]
*sigh*
```
const formatree = require('formatree');
```

**Comment by A1rPun** [comment]
Found it.. fix coming soon 👍

**Comment by A1rPun** [comment]
Fix -> [`9592e37`](https://github.com/A1rPun/formatree-cli/commit/9592e371d0028ba4192a6e9a1b17ad265681b050)
